conf/configure.fv3.wcoss2

```diff
@@ -0,0 +1,165 @@
+ifdef InNemsMakefile
+include $(TOP)/conf/configure.nems.NUOPC
+endif
+
+############
+# commands #
+############
+FC = ftn
+CC = cc
+CXX = CC
+LD = ftn
+
+#########
+# flags #
+#########
+DEBUG =
+REPRO =
+VERBOSE =
+OPENMP = Y
+HYDRO = N
+CCPP = N
+STATIC = N
+
+include       $(ESMFMKFILE)
+ESMF_INC    = $(ESMF_F90COMPILEPATHS)
+
+NEMSIOINC = -I$(NEMSIO_INC)
+NCEPLIBS = $(UPP_LIB) $(NEMSIO_LIB) $(G2_LIB4) $(G2TMPL_LIB) $(BACIO_LIB4) $(SP_LIBd) $(W3EMC_LIBd) $(W3NCO_LIBd) $(CRTM_LIB) $(PNG_LIB) $(JASPER_LIB) $(Z_LIB)
+
+##############################################
+# Need to use at least GNU Make version 3.81 #
+##############################################
+need := 3.81
+ok := $(filter $(need),$(firstword $(sort $(MAKE_VERSION) $(need))))
+ifneq ($(need),$(ok))
+$(error Need at least make version $(need).  Load module gmake/3.81)
+endif
+
+NETCDF_ROOT = $(NETCDF)
+INCLUDE = -I$(NETCDF_ROOT)/include
+
+FPPFLAGS := -fpp -Wp,-w $(INCLUDE)
+CFLAGS := $(INCLUDE)
+
+FFLAGS := $(INCLUDE) -fno-alias -auto -safe-cray-ptr -save-temps -ftz -assume byterecl -nowarn -sox -align array64byte
+
+CPPDEFS += -Duse_libMPI -Duse_netCDF -DSPMD -DUSE_LOG_DIAG_FIELD_INFO  -DUSE_GFSL63 -DGFS_PHYS -Duse_WRTCOMP
+CPPDEFS += -DNEW_TAUCTMAX -DINTERNAL_FILE_NML
+
+ifeq ($(HYDRO),Y)
+CPPDEFS +=
+else
+CPPDEFS += -DMOIST_CAPPA -DUSE_COND
+endif
+
+ifeq ($(32BIT),Y)
+CPPDEFS += -DOVERLOAD_R4 -DOVERLOAD_R8
+FFLAGS += -i4 -real-size 32
+else
+ifeq ($(REPRO),Y)
+FFLAGS += -i4 -real-size 64
+else
+FFLAGS += -i4 -real-size 64 -no-prec-div -no-prec-sqrt
+endif
+endif
+
+ifeq ($(REPRO),Y)
+FFLAGS += -qno-opt-dynamic-align
+CFLAGS += -qno-opt-dynamic-align
+else
+FFLAGS += -qno-opt-dynamic-align
+CFLAGS += -qno-opt-dynamic-align
+endif
+
+ifeq ($(MULTI_GASES),Y)
+CPPDEFS += -DMULTI_GASES
+endif
+
+FFLAGS_OPT = -O2 -debug minimal -fp-model source -qoverride-limits -qopt-prefetch=3
```

Comment:
Would you please add "-g -trackback" in FFLAGS_OPT? It was added to GFS.v16.0.17 and run in current opn. Please see https://github.com/ufs-community/ufs-weather-model/pull/874/files